ui/ozone/platform/x11/x11_native_display_delegate.cc

```diff
@@ -42,7 +49,31 @@ void X11NativeDisplayDelegate::Configure(const display::DisplaySnapshot& output,
                                          const display::DisplayMode* mode,
                                          const gfx::Point& origin,
                                          display::ConfigureCallback callback) {
-  NOTREACHED();
+  bool configure_success = false;
+
+  if (!mode) {
+    // This is a request to turn off the display.
+    configure_success = true;
+  } else {
+    // Check that |mode| is appropriate for the display snapshot.
+    for (const auto& existing_mode : output.modes()) {
+      if (existing_mode.get() == mode) {
+        configure_success = true;
+        break;
+      }
+    }
+  }
+
+  configure_callbacks_.push(
+      base::BindOnce(std::move(callback), configure_success));
+
+  // Start the timer if it's not already running. If there are multiple queued
+  // configuration requests then ConfigureDone() will handle starting the
+  // next request.
+  if (!configure_timer_.IsRunning()) {
```

Comment:
I can't understand why the timer needed. Can you explain more?

Reply:
I refereed to FakeDisplayDelegate::Configure and it adds delay to call callback.

Reply:
Here is the patch, which adds configuration delay to a fake display - https://codereview.chromium.org/2527263002 .

I still don't understand why you think X11 display needs it? It's not fake. You must use X APIs for configuration then and avoid having this kind of synthetic delays.

Reply:
Or leave it NOTIMPLEMENTED